### ISSUE TITLE
Wazuh app not loading on 7.8

### ISSUE THREAD

**Issue by j91321** [wazuh/wazuh-kibana-app#2337]
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 3.13.0   | 7.8.0     | ? |

**Description**
After upgrade to 7.8 Wazuh app in Kibana is stuck at, "Loading Elastic" page. I can't see any relevant logs/error messages in Kibana log.
  

**Steps to reproduce**

1. Go to Kibana
2. Click on Wazuh app

**Additional context**
I suspect it may be connected with X-Pack permissions, unfortunately Wazuh [documentation](https://documentation.wazuh.com/3.13/installation-guide/installing-elastic-stack/protect-installation/xpack.html) just says use `elastic` superuser for everything, which is not a good or secure guideline.

Current `kibana` user has `kibana_system` role and `wazuh_admin` role with `manage` and `manage_index_templates` cluster privileges, and `all` privilege to `.wazuh-*`, `wazuh-*`.

Also I'm using two Kibana nodes, but the plugin should be installed only on one of them. I don't think this should matter as far as I know wazuh doesn't use things like task manager which scale across multiple nodes.

I tried reinstalling the plugin.

**Comment by DeathsPirate** [comment]
I'm having this exact same issue, only I installed wazuh app on both kibana instances, one works the other doesn't and just stays on the "Loading Elastic" screen

**Comment by sadewale3** [comment]
having the same issue. It seems to be stuck on the loading screen. 

![wazuh](https://user-images.githubusercontent.com/46035289/86381983-9ff68300-bc5b-11ea-9dbf-8d57b8fb8634.PNG)

Sounds like this is a bug with 7.8.0 version of elk stack not compatible with the 3.13 versions of wazuh.

**Comment by juankaromo** [comment]
Hi @j91321 , @DeathsPirate , @sadewale3 and others...

We have performed several tests on x-pack, on 2 instances of Kibana, with different roles, and we have not detected any problem.

Please help us get to the source of this failure. We will check if there are any requests that are not being made correctly. From the development tools of the browsers press F12 (for chrome or firefox) and go to the Network tab. Once there, reload the page where the Wazuh app is located and check if there are any calls in red. That means the request has failed.

![image](https://user-images.githubusercontent.com/21195730/86478502-a44d9980-bd4a-11ea-8d17-35c5671e8755.png)

Any errors in the console would also be helpful.

On the other hand, could you run `systemctl status kibana -l` to see if there is an error? 
And `cat /usr/share/kibana/optimize/wazuh-logs/wazuhapp-plain.log`

Thank you very much for your help
Regards,

**Comment by pkoev** [comment]
@juankaromo managed to reproduce the behaviour on Mozilla Firefox 77.0.1(64) on Manjaro Linux. 
Same machine but Google Chrome and Chromium app works well.

**Comment by j91321** [comment]
I went to check steps provided by @juankaromo and noticed that the issue was not present in Incognito mode for Chrome. I took the following screenshot from the not working session in without incognito.
![wazuh-issue](https://user-images.githubusercontent.com/10012872/86587116-58d4fe80-bf89-11ea-9405-e72236cfc461.PNG)

Then I did "Clear browsing data..." in Chrome and now it seems that the Kibana app is working.
